index.html

```diff
@@ -42,7 +42,7 @@
 			 	<a href="#about">About</a>
 				<a href="#join">Join</a>
 			 	<a href="#projects">Projects</a>
-			 	<a href="#events">Events</a>
+			 	<a href="#events">Events<span class="label label-default">Sign up!</span></a>
```

Comment:
Not a big deal, but would it be better to add a space between Events and Sign up?